kustomize/overlays/dev/secrets/event-bridge-shard-operator-secrets.yaml

```diff
@@ -3,9 +3,11 @@ kind: SealedSecret
 metadata:
   annotations:
     sealedsecrets.bitnami.com/cluster-wide: "true"
-  name: event-bridge-shard-operator-secrets
+  name: addon-smart-events-operator-parameters
 spec:
   encryptedData:
+    EVENT_BRIDGE_MANAGER_URL: http://event-bridge:8080
```

Comment:
These need to be encrypted using Sealed Secrets. The SOP here tells you how to do it: https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-sop/blob/main/sops/update_secrets.asciidoc

Reply:
done